### PR TITLE
Added documentation for additional CLI functionality

### PR DESCRIPTION
**Pull request by claralebbos**

### CODE REVIEW

File: source/usage.rst [hunk 1]
```diff
@@ -23,6 +26,18 @@ Command Line Arguments
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Flag to show the output file in finder automatically
+ - The Python file will be saved with the same name as the output MP4 file (and so is defaulted to ``out.py``)
+
+-m, --manim
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Flag to only output generated python & manim code, and not create the animation
+ - The path defaults to ``out.mp4`` in the same directory the compiler is called. Use -o (description below) to specifiy the path.
```
Comment (mayanksharma3) [inline source/usage.rst]: Manim won't produce a mp4 file

File: source/usage.rst [hunk 2]
```diff
@@ -23,6 +26,18 @@ Command Line Arguments
 
  - The ``.manimdsl`` file to compile
 
+-f, --show_file_in_finder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Flag to show the output file in finder automatically
```
Comment (mayanksharma3) [inline source/usage.rst]: Finder is only MacOS

File: source/usage.rst [hunk 3]
```diff
@@ -23,6 +26,18 @@ Command Line Arguments
...
+ - The Python file will be saved with the same name as the output MP4 file (and so is defaulted to ``out.py``)
```
Comment (mayanksharma3) [inline source/usage.rst]: This flag doesn't really have anything to do with the python file naming (that's covered in ---python)

**Review by mayanksharma3 — CHANGES_REQUESTED** [review]

File: source/usage.rst (hunk 1 above)
Comment (claralebbos) [reply source/usage.rst]: updated

File: source/usage.rst (hunk 3 above)
Comment (claralebbos) [reply source/usage.rst]: updated

File: source/usage.rst (hunk 2 above)
Comment (claralebbos) [reply source/usage.rst]: updated

**Review by mayanksharma3 — APPROVED** [review]

**Pull request merged**